service/service-samples/sample-service-rest-redis-book/src/main/resources/service-book-application.properties

```diff
@@ -0,0 +1,18 @@
+#/*
```

Comment:
remove javasoc style comment and only add #

Reply:
done